edustripetech/edustripe_client_v1
# [Feature] : Implement home page responsiveness

#### What does this PR do?
- Implement home page mobile responsiveness
#### How should this be manually tested?
- Checkout to Feature/create-homepage branch and test locally.
- Visit / to view home page
- use chrome tools to check responsiveness
#### What are the relevant PT stories?
[173289686](https://www.pivotaltracker.com/story/show/173289686)
#### Screenshot
